i18n/en.json

```diff
@@ -11,14 +11,6 @@
 	"special-recordWizard-title": "Record Wizard",
 	"special-recordWizard-intro": "",
 
-	"apihelp-query+rwrecords-summary": "List all transcriptions of records made by a speaker in a given language.",
-	"apihelp-query+rwrecords-param-language": "Qid of the language's local wikibase item.",
-	"apihelp-query+rwrecords-param-speaker": "Qid of the speaker's local wikibase item.",
-	"apihelp-query+rwrecords-param-sort": "Property to sort by.",
-	"apihelp-query+rwrecords-param-dir": "Direction of the sort.",
-	"apihelp-query+rwrecords-param-format": "Element to get as output.",
-	"apihelp-query+rwrecords-param-limit": "How many transcriptions in total to return.",
-
```

Comment:
Shouldn't these be moved to a file named `i18n/api/en.json`?

Reply:
Drat! I created the files, but forgot to add them. Fixing...

Reply:
Should be fixed, please check.